OpenEdXMobile/src/main/java/org/edx/mobile/player/VideoPlayer.java

```diff
@@ -67,7 +68,7 @@ public VideoPlayer(Context context) {
     /**
      * Resets all the fields of this player.
      *
-     * @param context
+     * @param context Activity context
```

Comment:
not need, OR update to The `context of the current state of the application/object` cuz it is not activity-specific, it can be application context.